lms/djangoapps/grades/rest_api/v1/tests/test_gradebook_views.py

```diff
@@ -245,6 +245,40 @@ def test_course_grade_frozen(self):
             expected_data['grades_frozen'] = True
             assert expected_data == resp.data
 
+    def test_can_see_bulk_management_non_masters(self):
+        # Given a course without a master's track
+        # When getting course grading view
+        self.client.login(username=self.staff.username, password=self.password)
+        resp = self.client.get(self.get_url(self.course_key))
+
+        # Course staff should not be shown bulk management controls
+        assert resp.status_code == status.HTTP_200_OK
+        assert resp.data['can_see_bulk_management'] == False
+
+    @patch('openedx.core.djangoapps.enrollments.api.get_course_enrollment_details')
+    def test_can_see_bulk_management_masters(self, mock_course_enrollment_details):
+        # Given a course with a master's track
+        mock_course_enrollment_details.return_value = {'course_modes': [{'slug': 'not-masters'}, {'slug': 'masters'}]}
+
+        # When getting course grading view
+        self.client.login(username=self.staff.username, password=self.password)
+        resp = self.client.get(self.get_url(self.course_key))
+
+        # Course staff should be shown bulk management controls (default on for master's track courses)
+        assert resp.status_code == status.HTTP_200_OK
+        assert resp.data['can_see_bulk_management'] == True
+
+    @override_waffle_flag(waffle_flags()[BULK_MANAGEMENT], active=True)
```

Comment:
Does this mock a global override enable? Could we get a test with it overridden by course?

Reply:
The `waffle_flags()` grabs info from a bunch of different waffle/config settings, returning the rolled up output. Either a global or a course override would return true from there.

Reply:
^ This is specifically a `CourseWaffleFlag` on a course-by-course basis